### PR TITLE
dynamic mobile filtering

### PR DESCRIPTION
**Pull request by stephenmartin2951**
Changes to allow for dynamic filter options when browsing site via mobile.

### CODE REVIEW

**Review by kellenwiltshire — APPROVED** [review]
Looks Good!

**Comment by stephenmartin2951** [comment]
I just realized I left a console.log in there I will remove it in a bit

File: .gitignore [hunk 1]
```diff
@@ -4,13 +4,15 @@
 /node_modules
 /.pnp
 .pnp.js
+bun.lockb
```
Comment (kellenwiltshire) [inline .gitignore]: This should always be included in commits. It is the lock file, can you remove it from git ignore?

File: .gitignore [hunk 2]
```diff
@@ -4,13 +4,15 @@
...
 
 # testing
 /coverage
 
 # next.js
 /.next/
 /out/
+next-env.d.ts
```
Comment (kellenwiltshire) [inline .gitignore]: This too

**Review by kellenwiltshire — CHANGES_REQUESTED** [review]

**Review by kellenwiltshire — APPROVED** [review]

**Pull request merged**